README.md

```diff
@@ -66,27 +64,9 @@ swap-cli info
 
 Run `help` command for more details about available commands.
 
-### Remote client usage
```

Comment:
Why remove this here?

Reply:
All the docker usage now point to the same doc explaining both, it makes the readme a bit shorter and IMO the first use case now for remote client usage is within Docker, so I thought it makes sens like this.